go/engine/identify2_with_uid.go

```diff
@@ -130,12 +130,12 @@ func (i *identifyUser) isNil() bool {
 func loadIdentifyUser(g *libkb.GlobalContext, arg libkb.LoadUserArg, cache libkb.Identify2Cacher) (*identifyUser, error) {
 	ret := &identifyUser{arg: arg}
 	err := ret.load(g)
+	if ret != nil && ret.full != nil && cache != nil {
```

Comment:
yeah, you're right. the real fix was the swap of this check and the reset to `nil` (potentially) right beneath it